config.docker-template.php

```diff
@@ -27,6 +27,18 @@
 } else if (!empty($port)) {
     $CFG->wwwroot .= ":{$port}";
 }
+
+if (($themedevel = getenv('MOODLE_DOCKER_THEMEDEVEL')) !== false) {
+    $CFG->theme = $themedevel;
+    $CFG->devel_custom_additional_head = '<link rel="stylesheet" type="text/css" href="/theme/' . $themedevel . '/build/stylesheets/compiled.css" />';
```

Comment:
For some theme we have a preset or flavor. So we might have to customize the CSS filename too.

Reply:
That can be extended in `config-dev.php`; let's keep this in first, in line with https://github.com/liip-elearning/moodle-theme-template/pull/10/files#diff-5781ad689ba327f668ae99e7b13f1787R13